packages/server/__tests__/lib/grants-ingest.test.js

```diff
@@ -12,9 +12,12 @@ describe('processMessages', async () => {
 
     const serlializeGrantEvent = (newData = {}, prevData = {}) => {
         const eventData = { detail: { versions: { new: newData, previous: prevData } } };
-        if (newData !== {} && prevData !== {}) {
+        const newDataJSON = JSON.stringify(newData);
```

Comment:
I know this is a minor change, but we might want to leave it out of an ARPA specific PR.

Reply:
Agree in premise with avoiding scope-creep in PRs, but I'm generally fine with opportunistic unit test improvements as a result of someone being "in the neighborhood".

I'd have more of an issue if:
1. An out-of-scope change ends up generating a lot of discussion that holds up the delivery of everything else – certainly worth breaking out the change into a separate PR in that case.
2. There's an explosion of scope related to actual functional (runtime) code that makes it difficult to evaluate and/or test changes because so much is going on.

If there's an issue with these particular test updates that you think warrants more discussion  to discuss and/or think would benefit from scrutiny in a more dedicated PR, I can remove and submit separately, but if not I think I'm okay leaving as-is.